### DISCUSSION TITLE
Does anyone have a good gitignore for how to filter out unnesary quarto render files?

### DISCUSSION THREAD

**Discussion by usrbinr** [quarto-dev/quarto-cli#6072] [Q&A]
### Description

hi!

I use quarto and love it and publish a book to github pages but the outputs can be very verbose with the various supporting files.

Anyone have a good gitignore so that the the pages can still render the html but ignore unnecessary files?

**Comment by mcanouil** [comment]
What are you calling "supporting files" or "unnecessary files"?
Quarto already generates a `.gitignore` file which should be enough.
Additional things could be added depending on where you are publishing your website (as documented).
